src/main/java/FileClass.java

```diff
@@ -0,0 +1,46 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+import java.io.FileWriter;
+import java.io.IOException;
+
+
+public class FileClass {
+
+    public static void createFile(String filename) {
+        File f = new File(filename);
+        System.out.println("full path: " + f.getAbsolutePath());
+        System.out.println("file exists?: " + f.exists());
+        System.out.println("is Directory?: " + f.isDirectory());
+    }
+
+    private static void printFileHelper(String filePath) throws FileNotFoundException {
+        File f = new File(filePath); // create a File for the given file path
+        Scanner s = new Scanner(f); // create a Scanner using the File as the source
+        while (s.hasNext()) {
+            System.out.println(s.nextLine());
+        }
+    }
+
+    public static void printFileContents(String filename) {
+        try {
+            printFileHelper(filename);
+        } catch (FileNotFoundException e) {
+            System.out.println("File not found");
+        }
+    }
```

Comment:
Perhaps here you could catch the FileNotFoundException and then throw a custom DukeException such that your Duke class can handle the thrown Exception, and you wont need to handle this specific Exception at all reducing the scope of Exceptions thrown!